nextflow/accession.nf

```diff
@@ -89,27 +91,28 @@ process create_properties {
  * Accession VCFs
  */
 process accession_vcf {
-    clusterOptions {
-        log_filename = accession_properties.getFileName().toString()
-        log_filename = log_filename.take(log_filename.indexOf('_accessioning.properties'))
-        return "-g /accession/instance-${params.instance_id} \
-                -o $params.logs_dir/accessioning.${log_filename}.log \
-                -e $params.logs_dir/accessioning.${log_filename}.err"
-    }
+    clusterOptions "-g /accession/instance-${params.instance_id} \
+                    -o $params.logs_dir/${log_filename}.log \
+                    -e $params.logs_dir/${log_filename}.err"
 
     memory '8 GB'
 
     input:
     path accession_properties from accession_props
     val accessioned_filename from accessioned_filenames
+    val log_filename from log_filenames
 
     output:
     path "${accessioned_filename}.tmp" into accession_done
 
     """
     filename=\$(basename $accession_properties)
     filename=\${filename%.*}
-    java -Xmx7g -jar $params.jar.accession_pipeline --spring.config.name=\$filename
+    java -Xmx7g -jar $params.jar.accession_pipeline --spring.config.name=\$filename || \
+    # If accessioning fails due to missing variants, but the only missing variants are structural variants,
+    # then we should treat this as a success from the perspective of the automation.
+        [[ \$(grep -o 'Skipped processing structural variant' ${params.logs_dir}/${log_filename}.log | wc -l) \
+           == \$(grep -oP '\\d+(?= unaccessioned variants need to be checked)' ${params.logs_dir}/${log_filename}.log) ]]
```

Comment:
```suggestion
           == \$(grep -oP '\d+(?= unaccessioned variants need to be checked)' ${params.logs_dir}/${log_filename}.log) ]]
```

Reply:
This is confusing but actually fine, the backslash in the bash needs to be itself escaped so the Groovy parser doesn't interpret it as an escape character.